src/screens/welcome/install-referrer/onboardingPartner.ts

```diff
@@ -0,0 +1,127 @@
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import { Platform } from 'react-native'
+
+import Minilog from '@cozy/minilog'
+
+import { getInstallReferrer } from './androidPlayInstallReferrer'
```

Comment:
alias?

Reply:
Fixed in https://github.com/cozy/cozy-flagship-app/compare/e296534f408c5ca5dce5b8af44df9d6144165350..1b2e03a8f035457424a35b1ed39ad6fbb317bb73